### PR TITLE
PR_12

### PR DESCRIPTION
**Pull request by OksanaEfanova**
desc

### CODE REVIEW

**Comment by OksanaEfanova** [comment]
com

**Comment by OksanaEfanova** [comment]
comment

**Comment by OksanaEfanova** [comment]
comm2

**Comment by OksanaEfanova** [comment]
commen3

**Comment by OksanaEfanova** [comment]
A+

File: test_merge [hunk 1]
```diff
@@ -1 +1,3 @@
 test_merge
+errr
```
Comment (TestGitHub100500) [inline test_merge]: CR

**Comment by OksanaEfanova** [comment]
comment

**Comment by TestGitHub100500** [comment]
comment23

**Comment by TestGitHub100500** [comment]
comment24

**Comment by TestGitHub100500** [comment]
comment25

**Comment by TestGitHub100500** [comment]
comment26

**Comment by OksanaEfanova** [comment]
1

**Comment by TestGitHub100500** [comment]
2

**Comment by TestGitHub100500** [comment]
2

**Comment by TestGitHub100500** [comment]
3

**Comment by OksanaEfanova** [comment]
4

**Comment by OksanaEfanova** [comment]
5

**Comment by OksanaEfanova** [comment]
6

**Comment by OksanaEfanova** [comment]
7

**Pull request closed without merging**